### PR TITLE
support anonymous attendees, respect privacy

### PR DESCRIPTION
**Pull request by meonkeys**

### CODE REVIEW

**Comment by j2sol** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -38,6 +38,9 @@
         <p>
             <strong>Attendee Registration will not require the use of non-free software.</strong>
```
Comment (ex-nerd) [inline index.html]: Could just say " ... Registration is optional so you may attend without identifying yourself.  Nor does registration require ..."  Saves the extra paragraph below and doesn't sound quite so preachy.

Comment (meonkeys) [inline index.html]: That's pre-existing text (not part of my PR)

Comment (ex-nerd) [inline index.html]: Correct.  My recommendation was to amend the existing paragraph rather than add a new one.  Doesn't really matter now, I suppose.

Comment (meonkeys) [inline index.html]: Ah, gotcha.

I'm fine with whatever, I just want to run it by RMS when we've settled.

**Pull request merged**